platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,749 @@
+
+This benchmark compares pipeline run metrics between an AWS Batch compute environment with Fusion file system and fast instance storage enabled (**Fusion** group) and an identical AWS Batch compute environment using S3 storage without Fusion (**AWS S3** group).
+
+### Pipeline steps and computing resource requirements
+
+The nf-core/rnaseq pipeline involves several key steps, each with distinct computational requirements. Resource needs in this table are based on the `test_full` runs detailed previously:
+
+| **Pipeline step**                   | **Tools**                 | **Resource needs**           | **Description**                                                                                   |
+|-------------------------------------|---------------------------|------------------------------|---------------------------------------------------------------------------------------------------|
+| **Quality Control (QC)**            | FastQC, MultiQC           | Low-moderate CPU (50-200 % single-core usage), low memory (1-7 GB peak)     | Initial quality checks of raw reads to assess sequencing quality and identify potential issues.   |
+| **Read Trimming**                   | Trim Galore!              | High CPU (up to 700 % single-core usage), low memory (6 GB peak) | Removal of adapter sequences and low-quality bases to prepare reads for alignment.               |
+| **Read Alignment**                  | HISAT2, STAR              | Moderate-high CPU (480-600 % single-core usage), high memory (36 GB peak)       | Alignment of trimmed reads to a reference genome, typically the most resource-intensive step.    |
+| **Pseudoalignment**              | Salmon, Kallisto              | Moderate-high CPU (420 % single-core usage), moderate memory (18 GB peak)        | A faster, more accurate method of gene expression quantification than alignment using read compatibility.    |
+| **Quantification**                  | featureCounts, Salmon     | Moderate-high CPU (500-600 % single-core usage), moderate memory (18 GB peak) | Counting the number of reads mapped to each gene or transcript to measure expression levels.     |
+| **Differential Expression Analysis**| DESeq2, edgeR             | High CPU (650 % single-core usage), low memory (up to 2 GB peak )     | Statistical analysis to identify genes with significant changes in expression between conditions. |
+
+#### Overall run metrics
+
+The Fusion file system used with NVMe instance storage contributed to a 34 % improvement in total pipeline runtime and a 49 % reduction in CPU hours. 
+
+Total pipeline run cost: 
+- Fusion file system with fast instance storage: $ 34.90
+- Plain S3 storage without Fusion: $ 58.40
+
+![Run metrics overview](./_images/cpu-table.png)
```

Comment:
I find it strange that this graph sits under money values that aren't shown in the graph. Should they be plotted here too? Or can the graph me moved? Or the cost have a new section?

Reply:
Not a blocker

Reply:
Moved cost above with a small header. Thanks.